Source/DataSources/Entity.js

```diff
@@ -60,41 +60,50 @@ function createPropertyTypeDescriptor(name, Type) {
   });
 }
 
+/**
+ * @typedef {Object} Entity.ConstructorOptions
+ *
+ * Initialization options for the Entity constructor
+ *
+ * @property {String} [id] A unique identifier for this object. If none is provided, a GUID is generated.
+ * @property {String} [name] A human readable name to display to users. It does not have to be unique.
+ * @property {TimeIntervalCollection} [availability] The availability, if any, associated with this object.
+ * @property {Boolean} [show] A boolean value indicating if the entity and its children are displayed.
+ * @property {Property | string} [description] A string Property specifying an HTML description for this entity.
+ * @property {PositionProperty | CallbackProperty | Cartesian3} [position] A Property specifying the entity position.
+ * **** TODO ****
```

Comment:
What are these TODOs for? Also, CallbackProperty is officially supported here. The object needs to conform to the PositionProperty signature to work in all cases.  Please write up a separate issue so that we can either add a CallbackPositionProperty or something similar.

Reply:
Sorry, it's asking about the type of `orientation` and `viewFrom`.  Are they `Matrix4`? `Cartesian3`?  The docs aren't much help, as they just use the term to define itself ("Gets or sets the orientation" -- gee, thanks :D )

Reply:
re CallbackProperty, was that *isn't* officially supported?  I've been using it for a while but if that's wrong, we can remove it from the typing here and I'll just any-cast and cross my fingers until official support gets added.

Reply:
`viewFrom` is a `Cartesian3`. `Orientation` is a `Quaternion`

There are places where we call `PositionProperty.getValueInReferenceFrame`, such as path visualization and exportKml. So `CallbackProperty` is technically incorrect for now.  Turns out we have an issue for this already: #5534 so let's revert for now and maybe we can fix for 1.71.